pages/help.xml

```diff
@@ -96,21 +96,45 @@
         How deep does our integration go? Why should I care?
         What general functionality does the plugin include?
 				What functionality does the plugin omit? -->
+            <p>Plugin deploys ElectricFlow service as a single container or as a Docker swarm service based on whether given Docker endpoint is a stand-alone docker engine or swarm cluster manager. Plugin checks type of Docker endpoint at runtime and takes decision to deploy correct type of workload (container or swarm service) accordingly.</p>
 
```

Comment:
change "docker" to "Docker"
and "docker swarm" to "Docker Swarm". 
Change for this in the rest of the help file too.